src/ansys/dpf/core/mesh_info.py

```diff
@@ -0,0 +1,166 @@
+"""
+MeshInfo
+==========
+"""
+import ansys.dpf.core
+from ansys.dpf.core import server as server_module
+
+
+class MeshInfo:
+    """Represents the mesh information.
+
+    This class describes the available mesh information.
+
+    Parameters
+    ----------
+    mesh_info : ctypes.c_void_p, ansys.grpc.dpf.mesh_info_pb2.MeshInfo message
+
+     server : ansys.dpf.core.server, optional
+        Server with the channel connected to the remote or local instance.
+        The default is ``None``, in which case an attempt is made to use the
+        global server.
+
+    Examples
+    --------
+    Explore the mesh info from the model
+
+    >>> from ansys.dpf import core as dpf
+    >>> from ansys.dpf.core import examples
+    >>> fluent = examples.download_fluent_axial_comp()
```

Comment:
for the docstrings I created a specific mini version of the axial comp, `examples.fluid_axial_model()`. It directly returns a `DataSources` with just one `cas` and `dat`